sensing/pointcloud_preprocessor/launch/preprocessor.launch.py

```diff
@@ -40,19 +40,52 @@ def generate_launch_description():
     tf_output_frame_param = DeclareLaunchArgument("tf_output_frame", default_value="base_link")
 
     # set concat filter as a component
-    concat_component = ComposableNode(
-        package=pkg,
-        plugin="pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent",
-        name="concatenate_filter",
-        remappings=[("output", "points_raw/concatenated")],
-        parameters=[
-            {
-                "input_topics": LaunchConfiguration("input_points_raw_list"),
-                "output_frame": LaunchConfiguration("tf_output_frame"),
-                "approximate_sync": True,
-            }
-        ],
-    )
+    # if you want to use single node, set use_single_node = True
```

Comment:
It aims to control whether to separate sync node and concat node.
I make it to the rosparam in https://github.com/autowarefoundation/autoware.universe/pull/3312/commits/ee7bad22f6c809b370c58f81bf0fe412707158fc and https://github.com/autowarefoundation/autoware.universe/pull/3312/commits/a5d14078133647d4e2aa96f4949a8ce9575f1222.

Reply:
Actually, user may need to look at sample_sensor_kit_launch rather than the launcher in this package.
I will fix launcher in https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/62.